docs/mkdocs/docs/development/environment.md

```
@@ -0,0 +1,112 @@
+    container with a well-defined tool and runtime stack. This container can be used to run an application or to separate
+    tools, libraries, or runtimes needed for working with a codebase._
+
+Following docs section provides good overview of available features:
+
+- [Developing inside a Container]
+
+[Dev Containers]: https://containers.dev/
+[Install on Mac]: https://docs.docker.com/desktop/setup/install/mac-install/
+[Install on Windows]: https://docs.docker.com/desktop/setup/install/windows-install/
+[Install]: https://docs.docker.com/engine/install/
+[Docker Architecture]: https://docs.docker.com/get-started/docker-overview/#docker-architecture
```

Markdownlint (0.37.0)

Unused link or image reference definition: "docker architecture"
Link and image reference definitions should be needed

(MD053, link-image-reference-definitions)